src/main/kotlin/org/jitsi/nlj/dtls/DtlsUtils.kt

```diff
@@ -82,9 +82,9 @@ class DtlsUtils {
          * A helper which finds an SRTP protection profile present in both
          * [ours] and [theirs].  Throws [DtlsException] if no common profile is found.
          */
-        fun chooseSrtpProtectionProfile(ours: IntArray, theirs: IntArray): Int {
+        fun chooseSrtpProtectionProfile(ours: List<Int>, theirs: IntArray): Int {
```

Comment:
Using `List<Int>` for one and `IntArray` for the other feels a little weird here.  Could we use `Iterable` or `Collection` for both, maybe?

Reply:
Bleh. Yeah ok I guess it's fine as-is.

Reply:
Actually, `IntArray.asIterable()` isn't too bad, and doesn't make a copy.